MapboxMobileEvents/Categories/CLLocationManager+MMEMobileEvents.m

```diff
@@ -1,14 +1,25 @@
 #import "CLLocationManager+MMEMobileEvents.h"
-#import "MMEConstants.h"
+#import "../MMEConstants.h"
```

Comment:
All of these headers are flattened and would end up in the same directory so dropping `../` must work.